### PR TITLE
Feature/mr ceo/home carousel

### PR DESCRIPTION
**Pull request by AllStackDev1**
add carousel to the home page

### CODE REVIEW

File: src/components/home/components/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+                        <div className="carousel-text">
+                            <h3>{data.text}</h3>
+                        </div>
+                    </div>
+                ))}
+            </Carousel>
+        );
+    }
+}
+
```
Comment (Way2nnadi) [inline src/components/home/components/Carousel.js]: move all strings including class names to `constants.js` file

**Comment by Way2nnadi** [comment]
@MrCEO63 this looks good to me, fix moving strings to `constants.js` file and the build issue and this is good to merge down. Also, make sure to increment the patch version in the `package.json`. We use sem-ver(https://semver.org/) in this repo.

File: src/components/home/components/Home.js [hunk 2]
```diff
@@ -26,6 +27,7 @@ class Home extends React.Component {
                         </Col>
                     </Row>
                 </div>
+                <CarouselImages />
```
Comment (EngrEric) [inline src/components/home/components/Home.js]: remove and add this component to the `productList` component. @joshuaobinna  guide him properly as we want it to be.

**Review by EngrEric — COMMENTED** [review]
Looks good to me, just bump the version and make minor changes

File: src/components/home/components/Home.js (hunk 2 above)
Comment (joshuaobinna) [reply src/components/home/components/Home.js]: @MrCEO63 when a user navigates to the maket section,  the carousel should be seen just in between the navbar and the products displayed.  Dm me for more clarifications.

**Comment by joshuaobinna** [comment]
Code looks good so far. Stings, texts are placed in constants.js file

**Comment by Way2nnadi** [comment]
@EngrEric This should be merged down

**Review by EngrEric — APPROVED** [review]

**Pull request merged**